### ISSUE TITLE
Select and dropdowns not working on mobile browsers

### ISSUE THREAD

**Issue by luiskcs89** [lumapps/lumX#347]
Hi,

I've tried the selects and dropdowns, even the ones of the lumX website and they are not working on chrome on android and safari on iOS. Am I missing something or is it a real bug?

Thanks

**Comment by anoteboom** [comment]
experienced same issue.  the dropdown comes down, but i'm unable to tap on an item in the dropdown.
it seems that the event goes to the underlying object instead.

**Comment by gpolyn** [comment]
Similar: just cloned repo and serving locally I am unable to navigate to any of the four items in the main page header from Chrome Developer Tools mobile device views!

**Comment by SP2014** [comment]
Lumx has a great cross browser support, that I liked, but for mobile devices it is such a pain. I'm unable to use selects and dropdown. Please provide the fix.

**Comment by wfpaisa** [comment]
I already solved some of the problems

**Comment by playUruguay** [comment]
Hi!, I have the same issues, please, how I can fix? Thank you

**Comment by wfpaisa** [comment]
you can work with the latest version.
Clone the repo: git clone https://github.com/lumapps/lumx.git.

**Comment by magwest** [comment]
Still not working. Cloned the latest version

**Comment by melvincarvalho** [comment]
Not working for me either on android/chrome.  Desktop/chrome works fine.

**Comment by dholem** [comment]
I found removing the touchend in the lxDropdownMenu directive solved it for me.
```javascript
.directive('lxDropdownMenu', ['$timeout', function($timeout)
    {
        return {
            restrict: 'E',
            require: '^lxDropdown',
            templateUrl: 'dropdown-menu.html',
            transclude: true,
            replace: true,
            link: function(scope, element, attrs, ctrl, transclude)
            {
                var timer;

                ctrl.registerDropdownMenu(element);
                element.on('click', function(event)
                {
                    event.stopPropagation();

                    scope.$apply(function()
                    {
                        ctrl.toggle();
                    });
                });

                scope.$watch(function()
                {
                    return element.html();
                }, function(newValue)
                {
                    if (timer)
                    {
                        $timeout.cancel(timer);
                    }

                    timer = $timeout(ctrl.updatePositionAndSize, 150); // debounce
                });
            }
        };
    }])
```

**Comment by JuanSomniis** [comment]
Still not solution for select directive ?

**Comment by jmijmi** [comment]
Any news on this?

**Comment by matmkian** [comment]
We've just released LumX v1.0.0rc1. This new version fixes the issue, so I'm closing it.
